src/jobs/models/jobManager.js

```diff
@@ -200,20 +205,21 @@ function createResponse(jobId, jobBody, runId) {
     return response;
 }
 
-async function createJobRequest(jobId, runId, jobBody, dockerImage, configData) {
+async function createJobRequest(jobId, runId, reportId, jobBody, dockerImage, configData) {
     const jobTemplate = require(`./${configData.job_platform.toLowerCase()}/jobTemplate`);
     const jobName = util.format(JOB_PLATFORM_NAME, jobId);
     let maxVirtualUsersPerRunner = jobBody.max_virtual_users;
     const parallelism = jobBody.parallelism || 1;
     const environmentVariables = {
```

Comment:
as per our discussion on slack, let's add an env variable that will be sent to the runner:
`MIN_REQUIRED_VERSION` which will represent the minimum required version of predator-runner that can run the test. In the runner's logic, if the runner's version is less than the minimum requierd, it should postStats with a FAIL_STAGE with an appropriate error message

Reply:
Let us discuss it tomorrow, IMO it is not a healthy solution